vividus-plugin-mobile-app/src/main/java/org/vividus/mobileapp/model/ZoomType.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2019-2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.mobileapp.model;
+
+import org.openqa.selenium.Rectangle;
+
+public enum ZoomType
+{
+    IN, OUT;
+
+    private static final float CENTER_INDENT_COEFFICIENT = 0.1f;
+    private static final float TOP_INDENT_COEFFICIENT = 0.2f;
+    private static final float BOTTOM_INDENT_COEFFICIENT = 0.4f;
+    private static final float LEFT_INDENT_COEFFICIENT = 0.2f;
+    private static final float RIGHT_INDENT_COEFFICIENT = 0.2f;
+
+    @SuppressWarnings("MagicNumber")
+    public ZoomCoordinates calculateCoordinates(Rectangle zoomArea)
```

Comment:
I propose introducing a new class ZoomArea(Rectangle zoomArea) it will hide finger points calculation. 

ZoomArea zoomArea = new ZoomArea(Rectangle zoomArea);
var leftFingerPoint1 = zoomArea.leftFingerPoint1()
var leftFingerPoint2 = zoomArea.leftFingerPoint2()
var rightFingerPoint1 = zoomArea.rightFingerPoint1()
var rightFingerPoint2 = zoomArea.rightFingerPoint2()

in = 
leftFingerPoint1 -> leftFingerPoint2
rightFingerPoint1 -> rightFingerPoint2

out = 
leftFingerPoint2 -> leftFingerPoint1
rightFingerPoint2 -> rightFingerPoint1

Reply:
left and right fingers - that's something new 😄

Reply:
Didn't figure out anything better :) Just wanted to have something more meaningful than 1 and 2, yet it is an optional suggestion